diffsims/tests/test_utils/test_sim_utils.py

```diff
@@ -146,165 +140,32 @@ def test_uvtw_to_uvw(uvtw, uvw):
-    def test_ones(self):
-        data = np.ones((100, 100)) * 10
-        beta = bst_to_beta(data, 200000)
-        assert data.shape == beta.shape
-        assert (data != 0.0).all()
-
-
-class TestTeslaToAm:
-    def test_zero(self):
-        data = np.zeros((100, 100))
-        am = tesla_to_am(data)
-        assert data.shape == am.shape
-        assert (data == 0.00).all()
-
-    def test_ones(self):
-        data = np.ones((100, 100)) * 10
-        am = tesla_to_am(data)
-        assert data.shape == am.shape
-        assert (data != 0.0).all()
-
-    def test_known_value(self):
-        tesla = 1
-        am = tesla_to_am(tesla)
-        assert np.isclose(am, 795774.7155, rtol=1e-4)
-
-
-class TestAccelerationVoltageToVelocity:
-    @pytest.mark.parametrize(
-        "av,vel", [(100000, 1.6434e8), (200000, 2.0844e8), (300000, 2.3279e8)]
-    )  # V, m/s
-    def test_values(self, av, vel):
-        v = acceleration_voltage_to_velocity(av)
-        assert np.allclose(v, vel, rtol=0.001)
-
-
-class TestAccelerationVoltageToRelativisticMass:
-    def test_200kv(self):
-        mr = acceleration_voltage_to_relativistic_mass(200000)
-        assert np.isclose(1.268e-30, mr)
-
-
-@pytest.mark.parametrize(
-    "av,wl", [(100000, 3.701e-12), (200000, 2.507e-12), (300000, 1.968e-12)]
-)  # V, pm
-def test_acceleration_voltage_to_wavelength(av, wl):
-    wavelength = acceleration_voltage_to_wavelength(av)
-    assert np.allclose(wl, wavelength, rtol=0.001, atol=0.0)
-
-
-def test_acceleration_voltage_to_wavelength_array():
-    av = np.array([100000, 200000, 300000])  # In Volt
-    wavelength = acceleration_voltage_to_wavelength(av)
-    wl = np.array([3.701e-12, 2.507e-12, 1.968e-12])  # In pm
-    assert np.allclose(wl, wavelength, rtol=0.001, atol=0.0)
-
-
-class TestDiffractionScatteringAngle:
-    @pytest.mark.parametrize(
-        "mi,sa",
-        [
-            ((1, 0, 0), 9.84e-3),
-            ((0, 1, 0), 9.84e-3),
-            ((0, 0, 1), 9.84e-3),
-            ((2, 0, 0), 19.68e-3),
-            ((0, 2, 0), 19.68e-3),
-            ((0, 0, 2), 19.68e-3),
-        ],
-    )
-    def test_miller_index(self, mi, sa):
-        acceleration_voltage = 300000
-        lattice_size = 2e-10  # 2 Ångstrøm (in meters)
-        scattering_angle = diffraction_scattering_angle(
-            acceleration_voltage, lattice_size, mi
-        )
-        assert np.allclose(sa, scattering_angle, rtol=0.001)
-
-    def test_array_like(self):
-        # This should give ~9.84e-3 radians
-        acceleration_voltage = 300000
-        lattice_size = np.array([2e-10, 3e-10])
-        miller_index = (1, 0, 0)
-        scattering_angle = diffraction_scattering_angle(
-            acceleration_voltage, lattice_size, miller_index
-        )
-        assert len(scattering_angle) == 2
-        sa_known = np.array([9.84e-3, 6.56e-3])
-        assert np.allclose(sa_known, scattering_angle, rtol=0.001)
+def test_get_intesnities_params():
+    struct = make_structure()
```

Comment:
a cleaner syntax is to simply type `default_structure` as the argument of the function, it's defined in `conftest.py` and is the consistent way of parameterising tests. Then you just replace "`struct`" with "`default_structure`" ~ you couldn't have guessed this though as we clearly have both the "right" and the "wrong" way in the code in places